mindpype/containers.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, sess, value_type, val,
                 val = False
 
         self._ext_out = ext_out
-        self.data = val
+        self._data = val
```

Comment:
In cases where there is a setter method for the attribute, we can use the method within the `__init__` method. For example in this case, we would use `self.data = val`. The reason for this is that `val` could be passed in as anything when we are creating the object, so by using the setter, we explicitly check the content of `val` is appropriate and assigned to the `_data` attribute correctly.